src/main/proto/tech/figure/loan/v1beta1/loan_documents.proto

```diff
@@ -4,11 +4,13 @@ package tech.figure.loan.v1beta1;
 
 option java_multiple_files = true;
 
+import "google/protobuf/any.proto";
 import "tech/figure/util/v1beta1/document.proto";
 
 /*
 List of loan documents for a single loan
  */
 message LoanDocuments {
-  repeated tech.figure.util.v1beta1.DocumentMetadata document = 1; // Individual document metadata w/ pointer to actual document
+  repeated tech.figure.util.v1beta1.DocumentMetadata document    = 1; // Individual document metadata w/ pointer to actual document
+  map<string, google.protobuf.Any>                   metadata_kv = 2; // Any additional metadata
```

Comment:
shouldnt this be a `DocumentRecordingGuidnance`? 😕

Reply:
This protobuf isn't in the DART-affiliated package, so it should stay generic and independent of specific usecases, as alluded to in the PR description. This field follows the precedent of the rest of the `tech.figure` package, defining the most generic protobuf possible to allow usecases to provide any further fields as necessary.